CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Allow mmap to use new JDK-19 preview APIs in Apache Lucene 9.4+ ([#5151](https://github.com/opensearch-project/OpenSearch/pull/5151))
 - Add support for ppc64le architecture ([#5459](https://github.com/opensearch-project/OpenSearch/pull/5459))
 - Add support to disallow search request with preference parameter with strict weighted shard routing([#5874](https://github.com/opensearch-project/OpenSearch/pull/5874))
+- Adds support for minimum compatible version for extensions([#6003](https://github.com/opensearch-project/OpenSearch/pull/6003))
```

Comment:
Will this go into a 2.x release? If so, please move to the `[Unreleased 2.x]` section. Also be sure to add a space before the `(` or else @dblock will yell at you :)

Reply:
Thank you, added a beautiful space :)